docs/jwst/data_products/file_naming.rst

```diff
@@ -1,5 +1,10 @@
-File Naming Scheme
-------------------
+File Naming Schemes
+-------------------
+
+.. _exp_file_names:
```

Comment:
The underscores here (and for src_file_names) don't look like ASCII.  Is this OK?

Reply:
I think it might just be a weird way that it's being displayed. They are in fact ordinary ASCII underscores in the raw file and the links work.

Reply:
I think github might be interpreting the leading underscore to mean turn on underlining for the following words, so that's why it looks that way.